scripts/common.js

```diff
@@ -330,7 +330,7 @@ export async function fetchArticleIndex(offset) {
   if (index.done) return;
   let indexUrl;
   
-  if (isLocalhost()) {
+  if (isLocalhost() && window.localStorage.getItem('localIndex')) {
```

Comment:
What is this ?

Reply:
By default, local development now uses the live remote index. This local storage property allows to switch to the static local index - if you want that.

Reply:
interesting. i am not sure if the local index is needed altogether, which might let us cleanup the root a little bit. this is not a comment on this particular PR but something that i think we could consider... i'll file an issue.

Reply:
I found it increasingly inconvenient to work with the static local index.

Reply:
Removing special handling, see #465